src/main/resources/application.yml

```diff
@@ -67,6 +68,7 @@ vscode:
     version: unknown
   version: unknown
 ide-sidecar:
+  schema-fetch-error-ttl: 30
```

Comment:
Nit: can we put this lower, maybe just before `schema-fetch-retry` (since before it would be in proper sort-order)?